src/views/VariantsV2/components/PageContent/VariantsTable/index.tsx

```diff
@@ -170,15 +170,14 @@ const defaultColumns: ProColumnType[] = [
   {
     title: intl.get('screen.variants.table.participant.title'),
     tooltip: intl.get('screen.variants.table.participant.tooltip'),
-    dataIndex: 'studies',
     key: 'studies_participant',
-    render: (studies: IArrangerResultsTree<IVariantStudyEntity>) => {
-      const participantIds = studies?.hits?.edges
-        ?.map((study) => study.node.participant_ids || [])
-        .flat();
-      const nOfParticipants = participantIds.length;
-      if (nOfParticipants <= 10) {
-        return nOfParticipants;
+    render: (v: IVariantEntity) => {
+      const totalNOfParticipants = v.internal_frequencies?.total?.pc || 0;
```

Comment:
```suggestion
      const totalOfParticipants = v.internal_frequencies?.total?.pc || 0;
```
je comprends bien que c'est un reste d'avant mais qui abrege number en N ? je toruverai ca plus clair ainsi mais bon tu n'es pas oblige.